src/js/edu-benefits/utils/veteran.js

```diff
@@ -178,3 +184,130 @@ export function createVeteran() {
     }
   };
 }
+
+function convertFieldToValue(field) {
```

Comment:
I'd like us not to diverge too far from what we do for HCA: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/common/model/veteran.js#L700-L768